wordpress-mobile/WordPress-iOS
# Likes Notifications: add ID attributes to LikeUser entity.

Ref: #15662 

This adds new attributes to the `LikeUser` entity:
  - `likedSiteID`
  - `likedPostID`
  - `likedCommentID`

Bonus: corrected the spelling of `dateLikedString`.

To test:
- Verify the app builds and runs. 
- Verify it is using version 123 of the data model and the new attributes are present.
- Check `LikeUser` attributes to be sure they match up with WPKit `RemoteLikeUser` [properties](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/ad36aa868702606ef63512d11dc36bf50cd4726e/WordPressKit/RemoteUser%2BLikes.swift#L3).

## Regression Notes
1. Potential unintended areas of impact
N/A. Feature incomplete.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature incomplete.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature incomplete.

PR submission checklist:
